<launch>
  <node pkg="roboteq_driver" type="roboteq_manager" name="roboteq_manager">
    <param name="ppr" value="250000" />
    <param name="feedback_rate" value="5" />
    <param name="port" value="/dev/ttyACM0" />
  </node>
</launch>